lib/script-generator.js

```diff
@@ -54,7 +55,7 @@ ScriptGenerator.prototype.preamble = function () {
 
   var common = [
     '_script_name=`basename $0`',
-    'search_files=\'.\'',
+    'search_files=`find . -type f | grep -v \'./.git\'`',
```

Comment:
this scares me a bit. bash has a argument limit (I think it was 5000) and if you have something like api (or even mavis) anytime you deference this and use it as input to something it will break.  the alternative is to convert this into a bash array, and loop where you use this